source/03-components/accordion/accordion-item.twig

```diff
@@ -27,9 +27,6 @@
     <div
       class="c-accordion-item__drawer js-accordion-drawer"
       id="{{ section_id }}"
-      role="region"
-      aria-labelledby="{{ button_id }}"
-      aria-expanded="true"
```

Comment:
@dcmouyard I was planning on just swapping out `aria-expanded` for `aria-hidden` and leaving the `role` and `aria-labeledby`. Is there no reason to keep any of it? For example: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/

Reply:
@tomealter I was following the example of the [USWDS Accordion](https://designsystem.digital.gov/components/accordion/) component. I know `aria-expanded` is problematic here, but I’m not sure whether `role` and `aria-labelledby` are as well.

I think `aria-hidden` would work here as long as we’re still hiding the content via `display: none` somewhere. Out of the box we are with `slideToggle()`, but if folks remove that on a project then they would need to address this.